src/App.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from 'react';
+import nanoid from 'nanoid';
+import './App.scss';
+
+export default class App extends Component {
+
+  // setting initial state to match the schema on https://jsonresume.org/schema/
+  state = {
```

Comment:
the further I went the more I wish I had used react. loving this

Reply:
Same. but I'm barely learning to react. But this helps us see it's use and expand our exposure to it 👍